README.md

```diff
@@ -1,5 +1,8 @@
 <p align="center">
-  <img src="https://github.com/skypilot-org/skypilot/raw/master/docs/source/images/SkyPilot-logo-wide.png" alt="SkyPilot" width=55%/>
+  <picture>
+    <source media="(prefers-color-scheme: dark)" srcset="docs/source/images/skypilot-wide-dark-1k.png">
```

Comment:
Not sure if this relative path works with pypi, let me try with `test .pypi`.

Reply:
ahh good point. I can revert to the absolute URL if required

Reply:
Seems it will be broken if uploaded to the PyPi. Probably, we can use the original style of the link? 

https://test.pypi.org/project/skypilot/1.0.0.dev0/